ContentsquareModuleWrapper/Dummy.swift

```diff
@@ -0,0 +1,4 @@
+
+
+// Swift Package Manager needs at least one file to prevent a warning.
```

Comment:
the warning is in the client app ?

Reply:
The warning is on our package, 
anyway, the file is not needed as we'll add `ContentsquareBundle.bundle` later.

Reply:
actually it's needed :(